ui/admin/app/templates/scopes/scope/authenticate/method/oidc.hbs

```diff
@@ -12,7 +12,7 @@
     <p>{{t 'resources.auth-method.messages.pending.description'}}</p>
     <p>
       {{t 'resources.auth-method.questions.no-see-window'}}
-      <a href='#' role='button' {{on 'click' this.authMethod.authenticate}}>
+      <a href={{this.authURL}} target='_blank' rel='noreferrer noopener'>
```

Comment:
looks like this was intended to be a lik and not a button so refactored to match ui/desktop.